src/confluent/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '0.6.0'
+VERSION = '0.7.0'
```

Comment:
```suggestion
VERSION = '1.0.0b1'
```

Reply:
Please follow azure cli extension version schema and initialization as doc here: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#initialization